src/AssemblerHttpServer.cpp

```diff
@@ -326,7 +298,7 @@ void Assembler::writeNavigation(ostream& html) const
         {"Compressed assembly graph", "exploreCompressedAssemblyGraph"},
         });
     writeNavigation(html, "Help", {
-        {"Documentation", "docs/index.html"},
+        {"Documentation", "https://chanzuckerberg.github.io/shasta/index.html"},
```

Comment:
Let's take this out for now (leave in, commented out?). As is, this would point to a version of the documentation that is generally not consistent with the executable being run. Also, I don't want to introduce a reliance on GitHub Pages (per my other comment above).

Alternatively, we could leave this in, conditional to the `docs` directory being found in the expected location and containing `index.html`. I think it used to be that way at some point a long time ago. The check might have gone away as part of a restructuring of the navigation code.

Reply:
I will comment it out for now.